undofx/src/main/java/org/fxmisc/undo/UndoManagerFactory.java

```diff
@@ -14,118 +15,249 @@
 
 public interface UndoManagerFactory {
 
-    <C> UndoManager create(
+    /**
+     * Creates an {@link UndoManager} where changes will never be merged and every change emitted from the
+     * changeStream is considered to be a non-identity change.
+     *
+     * @param changeStream the {@link EventStream} that emits changes
+     * @param invert inverts the change
+     * @param apply applies the change (undo/redo)
```

Comment:
But there is useful information to add that is not captured by the type signatures.

What does it mean to do `apply(invert(c))`?

What should be expected to be emitted from `changeStream` after a call to `apply(c)`?

In the presence of `isIdentity`, what property should hold for `merge(c, invert(c))`?